File: 2206_divide_array_into_equal_pairs.cpp
class Solution {
public:
    bool divideArray(vector<int>& w) {
        sort(w.begin(), w.end());
        for (int v = 0; v < w.size(); v += 2) if (w[v] != w[v + 1]) return false;
        return true;
    }
};

File: 2206_divide_array_into_equal_pairs.java
class Solution {
    public boolean divideArray(int[] x) {
        Map<Integer, Integer> y = new HashMap<>();
        for (int z : x) y.put(z, y.getOrDefault(z, 0) + 1);
        for (int b : y.values()) if (b % 2 != 0) return false;
        return true;
    }
}
